ai_edge_torch/generative/examples/stable_diffusion/convert_to_tflite.py

```diff
@@ -47,8 +48,11 @@ def convert_stable_diffusion_to_tflite(
   diffusion = Diffusion()
   diffusion.load_state_dict(torch.load(diffusion_ckpt_path))
 
-  decoder = Decoder()
-  decoder.load_state_dict(torch.load(decoder_ckpt_path))
+  decoder_model = decoder.Decoder(decoder.get_model_config()).eval()
+  decoder_loader = autoencoder_loader.AutoEncoderModelLoader(
+      decoder_ckpt_path, decoder.TENSORS_NAMES
+  )
+  decoder_loader.load(decoder_model, strict=False)
```

Comment:
i think we can try removing `strict=False` here, since this is only needed when the model has non-trainable parameters (e.g. KV cache).

Reply:
Changed to `strict=True`